.github/workflows/phpunit.yml

```
@@ -2,6 +2,12 @@ name: PHPUnit
 
 on:
   pull_request:
+    paths:
+      - 'appinfo/**'
```

```suggestion
      - '.github/workflows/phpunit.yml'
      - 'appinfo/**'
```

Also adding this, but first want to test if it really skips